Makefile

```diff
@@ -32,13 +52,17 @@ $(ALL_SRC): ;
 generated_files: vendor
 	@./hack/update-codegen.sh
 
+go.mod:
```

Comment:
Will this always lead to init of project module?
Or this happens only when go.mod is missing?

Reply:
Yes, only if the file is missing.